### PR TITLE
update readme with instruction

### PR DESCRIPTION
**Pull request by henry60603**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,60 @@
...
+    - Edit perfeed/settings/.secrets.toml and add the necessary values 
+        - Github personal token, personal_access_token, is necessary. Please generate the token by going through the [Github doc](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+        - If you want to use OpenAI, a key is necessary. Please read the [OpenAI doc](https://help.openai.com/en/articles/4936850-where-do-i-find-my-openai-api-key)
+    - (Optional) Update the values in perfeed/settings/configs.toml if you'd like to change the default configurations.
+
+##  Execution 
+We use the jupyter notebook to do the summary. Here are the steps:
+1. Please go through the `perfeed/notebooks/weekly_summary_demo.ipynb` notebook with the example.
+
+2. Copy `perfeed/notebooks/weekly_summary_demo.ipynb` file to `perfeed/notebooks/weekly_summary.ipynb`
```
Comment (jzxcd) [inline README.md]: demo one is removed. only `weekly_summary.ipynb`

**Review by jzxcd — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,55 @@
-# Perfeed
-Personalized, AI assistant that runs with your engineering teams
+# 🚀 Perfeed
+A personalized AI assistant tailored to streamline engineering team workflows.
```
Comment (chihangwang) [inline README.md:2]: Let's make it clear that its main and only function is to summarize PRs to give a weekly summary. More to come later based on feature requests.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,55 @@
...
+## ⚙️ Getting Started
+
+### Prerequisites
+
+1. **Python**  
+   - Install Python 3.10 or higher.  
+   - **Recommendation**: Use a [Conda virtual environment](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) for managing Python versions.
+
+2. **Poetry**  
+   - Install **Poetry 1.8.3**, the package manager used to handle Python dependencies. Dependencies are listed in the `pyproject.toml` file.
```
Comment (chihangwang) [inline README.md:19]: Does poetry version matter?

**Review by chihangwang — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (henry60603) [reply README.md:19]: No, but I think the version is the latest one which I want to encourage them to use

File: README.md (hunk 2 above)
Comment (henry60603) [reply README.md:2]: add the feature section below

**Comment by henry60603** [comment]
would get back to fix the test pipeline

**Pull request merged**